test/test_ops.py

```diff
@@ -1257,6 +1257,8 @@ def get_vjp(cotangents, *primals):
                 'nn.functional.layer_norm'
             }
             if op.name in FUNCTORCH_HAS_FORMULA_BUT_NOT_PYTORCH:
+                self.assertFalse(op.supports_fwgrad_bwgrad,
+                                 f"{op.name} now supports forward over reverse without a decomposition")
```

Comment:
nit: Add something like "we should remove the decomposition" so there are some instructions for people who are not you and me